code/modules/clothing/masks/breath_warmer.dm

```diff
@@ -0,0 +1,21 @@
+/obj/item/clothing/mask/warmer
+	name = "warming mask"
+	desc = "A sterile mask designed to help prevent the spread of diseases."
```

Comment:
Shouldn't this description be different?

Reply:
yes